.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "0.2.0",
+    "configurations": [
```

Comment:
add files in .vscode to ignore list as well

Reply:
We are only including essential `.vscode` files. The list of essential files is there in `.gitignore`.